### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
### What?

I created the wireframe for a book store app using react. I also added the components and pages for this app.

### Why?

This is to help users keep track of books they have read with detailed information on attributes of the book.

### How?
- I created the app using react
- I started with the mobile first approach
- I created a hamburger icon for the mobile menu
- I created the components of the app
- I created pages inside the components
- I routed every link to their respective pages
- I moved all styles into one folder in the src directory
- I installed redux and redux toolkit
- I added a redux folder in the src folder
- I created a configureStore file in the redux folder
- I created a books and categories folder in the redux folder. These individual folders contain their respective JavaScript files which contain their respective reducers.
- I added some placeholder books through react-redux functionality
- I used PropTypes for prop validation
- I wrapped my app in Provider in index.js
- I used useDispatch for adding a new book
- I added 'remove' functionality to 'remove' button
- I used useDispatch to display 'under construction'
- I created a new book id inside the book.
- I refactored add and remove books actions.
- I added the API calls using AXIOS library.
- I added a new Slicer to the reducer.
- I added an Object.keys and a map function to receive the correct data from the books.
- I styled the page using [Zeplin Project](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) as a guide.

### CODE REVIEW

File: src/styles/book.css [hunk 1]
```diff
@@ -10,7 +10,41 @@
...
+  box-shadow: 0 0 3px #000;
+  color: blue;
+}
+
+.update-status-button:active {
+  background-color: orange;
+  border: 1px solid #000;
+  box-shadow: 0 0 3px blue;
+  color: #fff;
+}
```
Comment (tresorsawasawa) [inline src/styles/book.css]: - [x] Kindly make sure you improve your design a little bit to stick as much as possible to the template design(e.g.: **used fonts(family, size, weight,...) on your text(book title & other text, ), width & border_color of the book item block, alignment of the book item block text,...**. Make sure you check all details from the **[template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)** to match the design.

### Template Design
Screenshot from 2022-11-18 06-00-23

### Your Design
Screenshot from 2022-11-18 05-59-15

**But Good job with your design, it really looks perfect :+1: :+1: :100:**

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Haywayaheadshot 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitlow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/book.css (hunk 1 above)
Comment (Haywayaheadshot) [reply src/styles/book.css]: @tresorsawasawa Thank you for reviewing my project and appreciating the work done. I will improve on the styles asap

**Comment by Haywayaheadshot** [comment]
@tresorsawasawa Thank you for reviewing my project once again. I have made all the necessary adjustments.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Haywayaheadshot , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**Kindly check an optional comment I left under the review**_ :pray: 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 
-   Awesome UI. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
Hi @Qoosim Thank you for reviewing my project.

**Comment by Qoosim** [comment]
Hey @Haywayaheadshot, you're most welcome.
I hope you are doing great.

**Comment by Haywayaheadshot** [comment]
> Hey @Haywayaheadshot, you're most welcome.
> I hope you are doing great.

@Qoosim I am very well. I’m not sure you remember but you took me through how to install linters in my first week at Microverse. I am most grateful Qoosim.

**Pull request merged**